docs/api.rst

```diff
@@ -6652,7 +6566,7 @@ Querying periodic data
 
 Getting blockchain account data
 ===============================
-.. http:get:: /api/(version)/blockchains/(name)/accounts
+.. http:get:: /api/(version)/blockchains/(blockchain)/accounts
```

Comment:
We can also keep using name here. But if we do so we should rename the ones that use blockchain. You are only allowed to have one starter Pokemon ;)